pkg/engine/planner/physical/limit.go

```diff
@@ -10,9 +10,9 @@ type Limit struct {
 	id string
 
 	// Offset specifies how many initial rows should be skipped.
-	Offset uint64
+	Offset uint32
```

Comment:
nit: is there an advantage in naming the node limit field `Skip` (in `logical` pkg) but the physical planner field `Offset` here (`physical`) pkg?

Reply:
No, there is no reason for different names.